yuyo/components.py

```diff
@@ -2427,7 +2426,7 @@ class ComponentPaginator(ActionRowExecutor):
     This is a convenience class that allows you to easily implement a paginator.
     """
 
-    __slots__ = ("_authors", "_buffer", "_index", "_iterator", "_lock")
+    __slots__ = ("_authors", "_buffer", "_index", "_iterator")
```

Comment:
Todo: how does locking work for components when multiple ppl are using them and do we need to bring back these locks?